### ISSUE TITLE
dark willow plays very badly

### ISSUE THREAD

**Issue by andigortz** [arzon4dt/bot-experiment#8]
so I tried to find some mistakes why the dark will play badly as ally (dive to enemy towers, don't play on the lanes, etc), but play normally when they are enemies. Can anyone help with this issue?

**Comment by googyi** [comment]
> so I tried to find some mistakes why the dark will play badly as ally (dive to enemy towers, don't play on the lanes, etc), but play normally when they are enemies. Can anyone help with this issue?

I don't think it has anything to do with the bot-experiment code. The bot movement can be stupid because of the fallback valve c++ logic.

I saw phantom lancer standing at the fountain for no reason.
